keyboards/sofle_pico/config.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2024 Ryan Neff (@JellyTitan)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+/* Split Keyboard Stuff */
+#define EE_HANDS
```

Comment:
Not a fan of ee hands on ARM.  Split pin handedness is generally better.

Reply:
Thanks! I pulled out the ee_hands, and let it default to left hand as master. I updated the Readme to remove the separate flash for RH/LH eprom. I would have preferred the split pin method - but i'm running short on resources. :(